tools/bumper-bot/parser/parser.go

```diff
@@ -0,0 +1,48 @@
+package parser
+
+import (
+	"fmt"
+	"io/ioutil"
+
+	"github.com/pkg/errors"
+
+	"gopkg.in/yaml.v2"
+)
+
+type Component struct {
+	Name        string `yaml:"name"`
+	Url         string `yaml:"url"`
+	Commit      string `yaml:"commit"`
+	Release_tag string `yaml:"release_tag"`
```

Comment:
The idiomatic naming would be `ReleaseTag`

Reply:
I prefer this too but it breaks the parsing of the yaml. First char must be upper case and all the rest needs to be lower.